src/Components/Authentication/__tests__/Desktop/ModalManager.test.tsx

```diff
@@ -1,15 +1,15 @@
-import React from "react"
 import { mount, ReactWrapper } from "enzyme"
-import { ModalManager, ModalManagerProps } from "../../Desktop/ModalManager"
+import React from "react"
 import { LoginForm } from "../../Desktop/LoginForm"
+import { ModalManager, ModalManagerProps } from "../../Desktop/ModalManager"
 
 const getWrapper = (): ReactWrapper<ModalManagerProps> => {
   const wrapper = mount(
     <ModalManager
       submitUrls={{
         login: "/login",
         signup: "/sign_up",
-        reset_password: "/reset_password",
+        forgot_password: "/forgot_password",
```

Comment:
I believe the route is '/forgot' in Force, not sure about submission but worth double-checking

Reply:
Ah yeah, this is just a test and we grab the url from Force side but thanks for pointing out the inconsistency! Will update.

Reply:
Ohhh but you know what, Force uses the option `forgot` which will need to be changed throughout!